src/components/App/App.jsx

```diff
@@ -32,19 +34,51 @@ function App() {
     _id: "",
   });
   const [savedMovies, setSavedMovies] = useState([]);
+  const [isLoggedInLoading, setIsLoggedInLoading] = useState(true);
 
   // get default movies on page load
   useEffect(() => {
     setIsLoading(true);
-    api
-      .getInitialMovies(defaultMovies)
+    Ombdapi.getInitialMovies(defaultMovies)
       .then((data) => {
         setMovies(data);
       })
       .catch((err) => console.error(`Error fetching initial movies:`, err))
       .finally(setIsLoading(false));
```

Comment:
` .finally(setIsLoading(false));`


In `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well. 


```js
      .finally(() => setIsLoading(false));


```